src/lib/components/editor/EditorToolbar.svelte

```diff
@@ -41,13 +41,10 @@
     let isCommentBoxOpen = false;
     const { createNewThread } = commentStores;
 
-    let machineTranslations = machineTranslationStore.machineTranslations;
-    let machineTranslation = $machineTranslations.get(itemIndex);
-
     const showMachineTranslationRating =
         canEdit &&
         resourceContent.status === ResourceContentStatusEnum.TranslationEditorReview &&
-        $userIsEqual(machineTranslation?.userId);
+        $userIsEqual(resourceContent.assignedUser?.id);
```

Comment:
It looks like `canEdit` already checks if the user is assigned, so we should be able to get rid of this line.